draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -730,7 +730,7 @@ In some applications or transport protocols, it is desirable to put an SD-JWT in
 
 For such use cases, the SD-JWT SHOULD be transported as a single string. Key Binding MAY be achieved by signing the envelope JWT instead of including a separate Key Binding JWT in the SD-JWT.
 
-The following non-normative example shows an SD-JWT Presentation enveloped in a JWT payload:
+The following non-normative example shows an SD-JWT Presentation enveloped in a JWS payload:
```

Comment:
I don't totally love the suggested wording below but I think it's a "more correct" explanation of the content of the example. The text that was there previously doesn't seem quite right and JWT->JWS makes it maybe less correct. 

```suggestion
The following non-normative example shows the claims of an SD-JWT Presentation enveloped in a JWT:
```

Reply:
maybe make it simpler? `the claims of an SD-JWT presentation` took me a bit of time to understand.

```suggestion
The following non-normative example shows an SD-JWT Presentation enveloped in a JWT:
```

Reply:
That works. It's really showing the claims/payload of that JWT in which the SD-JWT is enveloped. But that's hard to describe. And what I'd suggested is pretty bad now that I read it again.